lib/galaxy/webapps/galaxy/controllers/admin_toolshed.py

```diff
@@ -1776,41 +1741,6 @@ def update_to_changeset_revision(self, trans, **kwd):
                                                         message=message,
                                                         status=status))
 
-    @web.expose
-    @web.require_admin
-    def view_tool_metadata(self, trans, repository_id, tool_id, **kwd):
```

Comment:
This is still used in the `Manage tools` view, if you click on a tool within a repository you'll get a 404.